src/promptflow-devkit/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # promptflow-devkit package
 
 ## v1.10.0 (Upcoming)
+- The `pf config set <key=value>` support set the folder where the config is saved by `--path folder` parameter,
```

Comment:
need to update this doc: https://microsoft.github.io/promptflow/how-to-guides/set-global-configs.html

Reply:
done